app/editor/src/features/content/form/hooks/useContentForm.ts

```diff
@@ -194,15 +194,11 @@ export const useContentForm = ({
 
   const resetForm = React.useCallback((values: IContentForm) => {
     // Reset form for next record.
-    const parsedDate = moment(values.publishedOn);
-    const updatedDate = parsedDate.add(1, 'second');
```

Comment:
Originally Scott wanted the published date to increment each time he clicked a new snippet.  I assume this behavior is no longer needed.